tests/dummy/app/templates/demo/select.hbs

```diff
@@ -99,4 +98,40 @@
 </p>
 {{code-snippet name="select.async-search.hbs"}}
 
+<h2>Select Header</h2>
+
+<div layout="column" layout-align="center center">
+  <div layout="column" layout-align="center center" style="height: 130px;">
+    {{! BEGIN-SNIPPET select.select-header }}
+  {{#paper-select
+    placeholder="Vegetable"
+    value=selectedVegetable
+    itemLabelCallback=vegetableLabelCallback
+    onChange=(action (mut selectedVegetable))
+    as |items|
+  }}
+    {{#paper-select-header class="demo-select-header"}}
+      {{input value=searchTerm
+              type="search"
+              placeholder="Search for a vegetable.."
+              class="demo-header-searchbox _md-text"
```

Comment:
I think it's safe to remove `_md-text` class.
It doesn't look like it is used for enything in this case. Also, it may confuse users.